tests/test_enterprise/api_client/test_discovery.py

```diff
@@ -401,12 +401,23 @@ def get_course_run(*args, **kwargs):  # pylint: disable=unused-argument
 
     @mock.patch('enterprise.api_client.discovery.course_discovery_api_client')
     @ddt.data(
-        (23, 'course-id', {'courses': {}}, False),
-        (45, 'fancy-course', {'courses': {'fancy-course': True}}, True),
-        (93, 'my-course', {'courses': {'my-course': False}}, False)
+        (23, 'course-v1:org+course+basic_course', {'courses': {}}, False, True),
+        (45, 'course-v1:org+course+fancy_course', {'courses': {'course-v1:org+course+fancy_course': True}}, True, True),
+        (93, 'course-v1:org+course+my_course', {'courses': {'course-v1:org+course+my_course': False}}, False, True),
+        (23, 'basic_course', {'courses': {}}, False, False),
+        (45, 'fancy_course', {'courses': {'fancy_course': True}}, True, False),
+        (93, 'my_course', {'courses': {'my_course': False}}, False, False)
     )
     @ddt.unpack
-    def test_is_course_in_catalog(self, catalog_id, course_id, api_resp, expected, mock_discovery_client_factory):
+    def test_is_course_in_catalog(
+            self,
+            catalog_id,
+            course_id,
+            api_resp,
+            expected,
+            is_run,
+            mock_discovery_client_factory
```

Comment:
Can we put the mock patch next to the function name? I notice we usually do that as it's easier to catch. For a moment I was wondering how in the world those booleans could be a `mock_discovery_client_factory`, but then I looked up.